client/app/collections/search.coffee

```diff
@@ -0,0 +1,35 @@
+{Filtered}  = BackboneProjections
+
+
+module.exports = class Search extends Filtered
+    constructor: (underlying, options = {}) ->
+        app     = require 'application'
+        @search = app.model.get 'filter'
+
+        options.filter = @filter.bind @
```

Comment:
As long as fat arrow create a closure for function execution, I assumed it isn't the best way for performance. But your comment let me do some search and I found that [bind is much slower than closure](http://stackoverflow.com/questions/17638305/why-is-bind-slower-than-a-closure). So I'll revert it, thanks!